meshtastic/tests/conftest.py

```
@@ -25,6 +28,17 @@
     from meshtastic.powermon.riden import RidenPowerSupply
     from meshtastic.powermon.stress import PowerStressClient
 
+
+def pytest_addoption(parser, pluginmanager):  # pylint: disable=unused-argument
```

Ruff (0.15.7)

[warning] Unused function argument: `pluginmanager`

(ARG001)